lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -4413,6 +4519,10 @@ void TFPartition::run() {
     if (!fn.isDefinition())
       continue;
     fns.push_back(&fn);
+    if (fn.getName() == "main") {
```

Comment:
Is there a canonical way to check if a function is the main function?

Reply:
There actually is.